scripts/requirements.txt

```diff
@@ -0,0 +1,4 @@
+docopt
+matplotlib
+numpy
+json
```

Comment:
```suggestion
```

Reply:
why?

Reply:
because it is a bug.  it is part of the python standard library.  if you try to install the requirements you will get an error

Reply:
OK, understood now. `requirements.txt` only to contain non-core libraries. Surprisingly to me, `json` is part of core.